<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>about</class>
 <widget class="QDialog" name="about">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>304</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>429</width>
    <height>304</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>429</width>
    <height>304</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>О программе</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string>Это проект научного многофункционального калькулятора SmallPAD.

Расчет любых выражений, построение графиков, интегрирование, решение уравнений.
Использует библиотеки: tiny-expr library и qcustomplot.

Обратная связь: Дмитрий Самосват
samosvat@yandex.ru</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
